<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1278</width>
    <height>834</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,1">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinAndMaxSize</enum>
      </property>
      <item>
       <widget class="QGraphicsView" name="graphicsView"/>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>GroupBox</string>
        </property>
        <widget class="QPushButton" name="button_reserAffichage">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>80</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>recalcul afficage</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBox_ligneObstacle">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>70</y>
           <width>72</width>
           <height>19</height>
          </rect>
         </property>
         <property name="text">
          <string>ligne obstacle</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QSpinBox" name="spinBox_Xrobot">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>110</y>
           <width>101</width>
           <height>22</height>
          </rect>
         </property>
         <property name="maximum">
          <number>4000</number>
         </property>
        </widget>
        <widget class="QSpinBox" name="spinBox_YRobot">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>110</y>
           <width>121</width>
           <height>22</height>
          </rect>
         </property>
         <property name="maximum">
          <number>4000</number>
         </property>
        </widget>
        <widget class="QLabel" name="label_XSouris">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>750</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_YSouris">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>750</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_tourLidar">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>150</y>
           <width>80</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>tour de lidar</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBox_Obstacle">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>70</y>
           <width>72</width>
           <height>19</height>
          </rect>
         </property>
         <property name="text">
          <string>obstacle</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1278</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
